server/src/definition/JinjaDefinitionProvider.ts

```diff
@@ -15,32 +15,36 @@ export class JinjaDefinitionProvider {
 
   constructor(private dbtRepository: DbtRepository) {}
 
-  onJinjaDefinition(document: TextDocument, packageName: string | undefined, jinja: ParseNode, position: Position): DefinitionLink[] | undefined {
-    if (packageName && this.isExpression(jinja.value)) {
-      const refDefinitions = this.refDefinitionFinder.searchRefDefinitions(document, position, jinja, packageName, this.dbtRepository.models);
-      if (refDefinitions) {
-        return refDefinitions;
-      }
+  onJinjaDefinition(
```

Comment:
This method can be refactored and looks similar to `DbtCompletionProvider` as the result. For definitionFinders we can have common interface.